.devcontainer/container_start.sh

```diff
@@ -1,8 +1,3 @@
-#!/bin/bash
-#set -e
+#!/bin/env zsh
```

Comment:
is `zsh` the default shell?

Reply:
Yes, is the default for [this image](https://github.com/PostHog/posthog/pull/7234/files#diff-13bd9d7a30bf46656bc81f1ad5b908a627f9247be3f7d76df862b0578b534fc6R12)